ngraph/core/reference/src/runtime/reference/interpolate.cpp

```diff
@@ -0,0 +1,309 @@
+//*****************************************************************************
```

Comment:
General comment, please add comments to the code.

Reply:
Comments are added.